sentry_sdk/transport.py

```diff
@@ -202,12 +254,50 @@ def _send_request(
                     response.data,
                 )
                 self.on_dropped_event("status_{}".format(response.status))
+                record_loss("network_error")
         finally:
             response.close()
 
     def on_dropped_event(self, reason):
         # type: (str) -> None
-        pass
+        return None
+
+    def _fetch_pending_client_report(self, force=False, interval=60):
+        # type: (bool, int) -> Optional[Item]
+        if not self.options["send_client_reports"]:
+            return None
+
+        if not (force or self._last_client_report_sent < time.time() - interval):
```

Comment:
```suggestion
        if not force or time.time() - interval < self._last_client_report_sent:
```

easier to read this way imo

Reply:
i'm never sure what the prioritization is, would keep it explicit.